### PR TITLE
docs: cleanup README and add more background

### PR DESCRIPTION
**Pull request by stvstnfrd**
This is best reviewed commit-by-commit.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,8 +1,28 @@
-In Video Quiz XBlock |BS| |CA|
-==============================
+In Video Quiz XBlock
+====================
 
 This XBlock allows for edX components to be displayed to users inside of videos at specific time points.
```
Comment (sarina) [inline README.rst:4]: Any component or just problem/capa components? (or are all components implied to be problems?)

File: README.rst [hunk 2]
```diff
@@ -1,8 +1,28 @@
...
+added to the same Unit as itself.
+
+The XBlock is then configured (see below) to map timestamps (in seconds)
+to Component IDs (the Problems), along with the Video Component ID.
+
+Whenever video playback reaches a configured timestamp, the video is
+paused and the corresponding Problem Component is overlayed on top of
+the Video.
+
+The user can answer problem, though it is not required.
```
Comment (sarina) [inline README.rst:19]: nit: answer the problem

question: can you make it required?

File: README.rst [hunk 3]
```diff
@@ -26,22 +46,61 @@ advanced settings.
...
+    ``cf5557d7eee9408786116f9fead8b2ba``
+``Problem Timemap``: A JSON dictionary with:
+    - timestamps (in seconds, as a string) as keys
+    - a problem component ID as the value, eg:
+    ``{"5": "10fd296e593e404ba999d78c6a76db84"}``
+
+This would display the Problem ``10fd29...`` 5 seconds into Video ``cf5557...``.
+
+TODO: This should be further fleshed out to explain how to get these
+values, etc.
```
Comment (sarina) [inline README.rst:62]: Can you crib wholesale from that blog post draft?

File: README.rst [hunk 4]
```diff
@@ -26,22 +46,61 @@ advanced settings.
...
-License
--------
+With a small amount of code, we're able to leverage the functionality of
+existing problems and the video player into a fairly elegant solution.
 
-The In Video Quiz XBlock is available under the AGPL Version 3.0 License.
+But as it depends upon expected behavior and display in the LMS,
+it requires a specific configuration
+and the behavior is vulnerable to breaking as the LMS experience changes
+(eg, the conversion to the Learning MFE from the Legacy Courseware).
```
Comment (sarina) [inline README.rst:85]: Does this block work in the learning MFE?

Comment (sarina) [reply README.rst:85]: Would this block work with any video player or just the current default Open edX video player? (I ask as I'm proposing a blended project to overhaul or replace the current player)

File: README.rst (hunk 2 above)
Comment (stvstnfrd) [reply README.rst:19]: You can _not_ force it to be required; this was an intentional product choice.

File: README.rst (hunk 1 above)
Comment (stvstnfrd) [reply README.rst:4]: Technically anything should work, though only used in practice with problem/capa.

File: README.rst (hunk 4 above)
Comment (stvstnfrd) [reply README.rst:85]: It works in the MFE _now_; it didn't until I fixed it/added support last week.

Comment (stvstnfrd) [reply README.rst:85]: In theory, it could be made to work.
I think you just need a basic play/pause API to interact with the video player.

**Pull request merged**